js/modules/networkgraph/networkgraph.src.js

```diff
@@ -754,14 +757,18 @@ addEvent(
                                 chart.container,
                                 'mousemove',
                                 function (e) {
-                                    return point.series.onMouseMove(point, e);
+                                    return point &&
+                                        point.series &&
+                                        point.series.onMouseMove(point, e);
```

Comment:
Since point.series is being used multiple times, should we consider creating a variable instead and access it from there ?

Reply:
In general - yes. However, `point` can be undefined in a first place, so this would throw an error.